README.md

```diff
@@ -171,9 +171,87 @@ const config = {
 module.exports = config;
 ```
 
+## Biome Configuration
+
+`@technance/code-style` provides a [Biome](https://biomejs.dev/) configuration for ultra-fast linting and formatting. Biome is a performant alternative to ESLint and Prettier, written in Rust.
+
+### Installation
+
+First, install Biome as a development dependency:
+
+```bash
+pnpm add @biomejs/biome -D
+```
+
+```bash
+npm install --save-dev @biomejs/biome
+```
+
+```bash
+yarn add -D @biomejs/biome
+```
+
+### Setup
+
+Create a `biome.json` or `biome.jsonc` file in your project root that extends the shared configuration:
+
+```json
+{
+    "extends": ["@technance/code-style/biome"],
+    "linter": {
+        "enabled": true
+    },
+    "formatter": {
+        "enabled": true
+    }
+}
+```
+
+### Usage
+
+You can run Biome commands directly:
+
+```bash
+# Check your code (lint + format check)
+npx @biomejs/biome check .
+
+# Format your code
+npx @biomejs/biome format --write .
+
+# Lint your code
+npx @biomejs/biome lint .
+
+# Apply safe fixes
+npx @biomejs/biome check --write .
+```
+
+Alternatively, add these scripts to your `package.json`:
+
+```json
+{
+    "scripts": {
+        "lint": "biome check .",
+        "lint:fix": "biome check --write .",
+        "format": "biome format --write ."
+    }
+}
+```
```

Comment:
Let's drop this part as it is mostly a documentation for the Biome itself, not necessarily related to our code-style package.